src/forms/BenefitsTable.js

```diff
@@ -84,44 +84,44 @@ const SNAPBenefitRow = function( props ){
     return (
       <Table.Row>
         <Table.Cell style={rowHeaderStyle}>SNAP</Table.Cell>
-        <Table.Cell textAlign='right'>${SNAPBenefitCurrent} / year</Table.Cell>
-        <Table.Cell textAlign='right'>${SNAPBenefitFuture} / year</Table.Cell>
-        <Table.Cell textAlign='right'>{ getSignSymbol(SNAPDiff) } ${Math.abs(SNAPDiff)} / year</Table.Cell>
+        <Table.Cell textAlign='right'>${SNAPBenefitCurrent/12} / month</Table.Cell>
```

Comment:
Correct math! Values aren't rounded, though. You could round down here, but I suggest looking at where the numbers are gotten in the first place and seeing if you can change the math there instead!